exchanges/subscription/list.go

```diff
@@ -55,6 +55,17 @@ func (l List) Clone() List {
 	return n
 }
 
+// Authenticated returns only Authenticated subscriptions
+func (l List) Authenticated() List {
```

Comment:
Match the func elsewhere, or wait until the other one is merged 🤙

Reply:
Fixed aa10abb3c